connection-limit/connection_limit_user.c

```diff
@@ -34,50 +34,22 @@ static const char *__doc__ =
         "Connection limit incoming TCP connections using XDP";
 
 static int ifindex;
-static char prev_prog_map[1024];
 
-int map_fd[MAP_COUNT];
+int conn_count_map_fd = -1, tcp_conns_map_fd = -1, conn_info_map_fd = -1;
```

Comment:
can we retain map_fd[] array and have enums for each index like, CONN_COUNT, TCP_CONNS & CONN_INFO so that its extensible ?